src/dota2/decoders.ts

```diff
@@ -258,3 +259,12 @@ export function decodeWearable(rawWearable: any) {
   });
   return iWearableList as IWearbleItem[];
 }
+
+export function decodeEvents(rawEvents: Array<any>) {
+  return rawEvents.map((event) => {
+    return { 
+      gameTime: getAttr(event, 'game_time'),
+      eventType: getAttr(event, 'event_type'), 
```

Comment:
I'm currently only returning two properties of the entire event object because I'm not sure how to disambiguate between a rune or roshan event. Any ideas? Should they return different interfaces?